pkg/handler/repositories.go

```diff
@@ -673,14 +684,19 @@ func (rh *RepositoryHandler) enqueueUpdateEvent(c echo.Context, response api.Rep
 	}
 }
 
+func (rh *RepositoryHandler) CheckSnapshotForRepo(c echo.Context, orgId string, snapshotParam *bool) error {
```

Comment:
I know it was there already, but can orgId be removed as a parameter? I don't see it being used